src/constants/task.ts

```diff
@@ -0,0 +1,26 @@
+export enum TaskOperation {
+  ADD = 'Add',
+  EDIT = 'Edit',
+}
+
+export const TaskValidationLimits = {
+  TITLE_MIN_LENGTH: 3,
+  DESCRIPTION_MIN_LENGTH: 10,
+};
+
+export const TaskMessages = {
```

Comment:
These messages are still here. Should be either hardcoded into UI components or loaded from locale files.

Reply:
removed these too